script/foreman-debug

```diff
@@ -149,6 +144,22 @@ VERBOSE=0
 DEBUG=0
 UPLOAD=0
 UPLOAD_DISABLED=0
+UPLOAD_URL='rsync://theforeman.org/debug-incoming'
+UPLOAD_CMD='rsync "${TARBALL}" "${UPLOAD_URL}"'
+UPLOAD_USAGE_MSG="\
+You may want to upload the tarball (with -u) to our public server via rsync.
+This is a write-only directory (readable only by Foreman core developers)
+Please note that the rsync transmission is UNENCRYPTED."
+UPLOAD_UNSET_MSG="\
+To upload a tarball to our secure site, please use the -u option."
+UPLOAD_SUCCESS_MSG="\
+The tarball has been uploaded, please contact us on our mailing list or IRC
```

Comment:
Mind changing this to `https://community.theforeman.org`?

Reply:
The Messages now read:
```
+UPLOAD_SUCCESS_MSG="\
+The tarball has been uploaded, please contact us on community.theforeman.org
+or on IRC/Matrix (#theforeman), making sure you reference the following URL:
+
+    http://debugs.theforeman.org/"
+UPLOAD_FAIL_MSG="\
+The tarball could not be uploaded, please upload it to an alternate location
+and contact us on community.theforeman.org or on IRC/Matrix (#theforeman),
+making sure you reference that URL."
```

Reply:
https://theforeman.org/support.html#IRClivechat or https://theforeman.org/support.html#Matrix.orglivechat could be linked to for IRC/Matrix. Or just https://theforeman.org/support.html in general because it also refers to the forum.